utility_scripts/compare_models.py

```diff
@@ -659,13 +681,43 @@ def _create_boxplot_for_column(self, plot_figure, column_to_plot, model_1_data,
             handlelength=0
         )
 
+    def _create_violinplot_for_column(self, plot_figure, column_to_plot, model_1_data,
+                                      model_1_n_samples, model_2_data, model_2_n_samples, title):
+        ax = plot_figure.add_subplot(self.n_rows, self.n_cols, self.index)
+        sns.violinplot(
+            data=pd.concat([model_1_data, model_2_data]),
+            x=BINARIZED_LABEL,
```

Comment:
Maybe it's possible to make the xlab equal-ish to boxplot? (so benign/pathogenic instead of 0/1?)

If it's too much work just leave it as is, if it's a quick fix might be a nice finishing touch.